docs/user-guide/analyses/phenotype-scores.rst

```diff
@@ -38,11 +38,11 @@ This is a non-parametric test that compares the medians of the two classes to de
 >>> class2 = [4, 5, 3, 4, 3, 3, 3, 4, 4, 5, 5, 2, 3, 0, 3, 5, 2, 3]
 >>> r = stats.mannwhitneyu(x=class1, y=class2, alternative = 'two-sided')
 >>> p_value = r.pvalue
->>> float(p_value)
-6.348081479150902e-06
+>>> float(p_value) # doctest: +ELLIPSIS
```

Comment:
The doctest runner seems to recognize the `...` even without `#doctest: +ELLIPSIS` marker, and the test passes here. Does the same happen on your end? If yes, pls remove `# doctest: +ELLIPSIS`.